java/com/facebook/soloader/SysUtil.java

```diff
@@ -73,6 +80,61 @@ public static void deleteOrThrow(File file) throws IOException {
     }
   }
 
+  /**
+   * Return an list of ABIs we supported on this device ordered according to preference.
+   * @param context
+   * @return Ordered array of supported ABIs, the ABI nativeLibrary used is top priority.
+   */
+  public static String[] getSortedSupportedAbisByNativeLibrary(Context context) {
```

Comment:
Instead of adding a new API, change LollipopSysdeps.getSupportedAbis().
The reason the logic is isolated into its own class is to limit class verification failures. Having an unverifiable method such as this one would fail verification for the whole class and slow down already-slow pre-L phones even more :)

Reply:
Thank you for your reply!!
Yeah, i agree with you. I have changed LollipopSysdeps and removed getSortedSupportedAbisByNativeLibrary.
BTW, Could you please explain what the following sentence means?👇 (Sorry, my English is poor..hah

> minor code style/organization things inline.